### ISSUE TITLE
[Privacy] Mixpanel use phase out?

### ISSUE THREAD

**Issue by Filbuntu** [wireapp/wire#259]
A friend of mine can't trust Wire because of the Mixpanel track use, he argues it sends data before he can opt-out (by the way, is the user asked to be opt-in/out upon installation?) and Signal & Threema are developed well without such tracker modules. 
Here are some links regarding the discussion he found:
https://www.reddit.com/r/privacytoolsIO/comments/8r44oz/signal_vs_wire/
https://mobile.twitter.com/wire/status/988327728486211585
[Github Wire: Replaced Localytics with Mixpanel ](https://github.com/wireapp/wire-webapp/commit/0bfbe8e6f877f17f023fcbef7a1fe013df84d9f7)
[Wire Privacy Whitepaper](https://wire-docs.wire.com/download/Wire+Privacy+Whitepaper.pdf)
In German, https://www.deepl.com/translator.html can be used for translation:
https://www.androidpit.de/datenschutz-messenger-wire-tracking-code-sendet-daten-in-die-usa?sac=true#comment2605565
https://www.kuketz-blog.de/messenger-wire-beinhaltet-mixpanel-app-tracking/
https://www.kuketz-blog.de/messenger-wire-reagiert-auf-mixpanel-tracking/

Will there be a possibility to phase out the Mixpanel use? As Wire wants to stand for security and privacy and its competitors can do without trackers, I think my friend's concerns need to be considered.

**Comment by Filbuntu** [comment]
I would appreciate an answer. Thanks!

**Comment by marcoconti83** [comment]
Hi @Filbuntu ,
Wire used Mixpanel in the past, but has been discontinued by fall 2018 (different platforms removed it at different times). Right now none of the Wire clients (iOS, Android, web and desktop versions) uses Mixpanel anymore.

I hope this answers your question!

**Comment by Filbuntu** [comment]
Hi @marcoconti83 !
Thanks a lot for the encouraging feedback! Was Mixpanel just discontinued or replaced by another tracker?

**Comment by marcoconti83** [comment]
Mixpanel was removed and we did not replace it with any other tracker yet. We are now relying on the install/session stats provided by the native stores (Google Play Store, Apple App Store) and Hockey/Raygun for crash logs. 

We might consider re-introducing a self-hosted solution (on our backend) for analytics in the future, because analytics help us understand what part of the product are the most used and make them better/more reliable. 

Regarding user choice, the app asks for your consent (opt-in) after every login on a new device, so if you don't opt-in, the analytics/crash reporting services will not receive any data.
